release/release-process.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Release Process
+---
+<-- TODO -->
+# How to Update Version
+
+<-- TODO -->
+# How to Tag
+
+<-- TODO -->
+# Test Process
+
+<-- TODO -->
+# Build Process
+
+
+# Publish to Maven Central
+
+## Account Setup
+Following [sonatype.org](http://central.sonatype.org/)
+
+1. Request access to publish to com.twitter by creating account and requesting access in https://issues.sonatype.org
+2. Twitter contact needed to confirm access [see example](https://issues.sonatype.org/browse/OSSRH-22297)
+3. Install local GPG/PGP to sign files by following [sonatype instructions](http://central.sonatype.org/pages/requirements.html#sign-files-with-gpgpgp) 
+4. Publish public key [see example](http://central.sonatype.org/pages/working-with-pgp-signatures.html#distributing-your-public-key)
+5. Confirm public key is published [see example](http://pgp.mit.edu/) note leading ```0x``` required for hex keys)
+
+# Release Steps
+Follow [Manual Staging Bundle Creation and Deployment](http://central.sonatype.org/pages/manual-staging-bundle-creation-and-deployment.html) by signing files manually, steps summarized below.
+
+Three artifacts are required for each version: ```heron-api```, ```heron-storm```, and ```heron-spi```
+Example POM template files required located in ```heron/release/maven-central/```
+
+<-- TODO refer to release "build process" instructions, not download  -->
+<-- TODO update for "heron-spi" artifact when available -->
+### Step 1 - Download and install ```heron-api-install-{VERSION}-{DIST}.sh``` from https://github.com/twitter/heron/releases
+
+Example for version ```0.14.0``` installed using ```--user```
+```
+$ wget https://github.com/twitter/heron/releases/download/0.14.0/heron-api-install-0.14.0-darwin.sh
+$ chmod +x heron-api-install-0.14.0-darwin.sh
+$ ./heron-api-install-0.14.0-darwin.sh --user
+```
+Jars are installed into ```~/.heronapi```
+
+### Step 2 - Create temp folder for each ```heron-api``` and ```heron-storm``` artifact 
+Example, create folder named ```heron-api-bundle-0.14.0```
+
+Artifact bundle must contain:
+
+1. {artifact-version}.jar
```

Comment:
@billonahill example of format?  i'm not sure how to proceed on this one, these are artifact jars generated by the Bazel release, thanks.